index.html

```diff
@@ -2331,7 +2331,14 @@ <h3>
 The <a>DID method</a> specification MUST specify how a client can update a
 <a>DID document</a> on the <a>DID registry</a>, including all cryptographic
 operations necessary to establish proof of control, <em>or</em> state that
-updates are not possible.
+updates are not possible. 
+        </p>
+
+        <p>
+          An Update to a DID is any change in the data used to produce a DID Document. 
+          DID Method Implementers are responsible for defining what constitutes an update, 
+          and what properties of the DID Document are supported by a given DID Method. 
+          There may be cases where an "Update" occurred but the DID Document representation does not change.
```

Comment:
```suggestion
          There might be cases where an update occurred but the <a>DID document</a> representation does not change.
```

We probably don't have `representation` defined in the terminology and should define it... can be done in a different PR.